encryption/README.md

```diff
@@ -0,0 +1,52 @@
+# Encryption
+
+## What is it?
+
+Encryption involves complex mathematics to obscure data so that it is almost impossible to read without the correct decryption keys or passwords. End-to-end encryption is commonly used when transferring information that needs to be kept secret over a public system such as the internet.
+
+Critically, Encryption provides the following benefits:
+
+**Non-repudiation** - You can be sure that only the intended sender sent that message. Nobody else.  
+**Integrity** - The message has not been changed on the way to you by someone malicious.  
+**Confidentiality** - Nobody except you and the sender can understand the message.
+
+
+## Why is it important?  
+Without encryption the internet loses a lot of it's functions...
+
+1. Home Finance
+Without being able to prove who you are without risking your life savings, banks are unable to allow internet banking from within the UK.
+
+2. Online Retail
+Without being able to transfer payment details online securely, online retailers no longer are able to sell to UK citizens without risking high levels of fraud. Your bank will also want to avoid insecure online payments as this exposes them to fraud.
```

Comment:
As shown above, I would suggest writing these as hypotheticals - 'would no longer be able', 'would want to avoid', 'would expose them to fraud' etc.